app/views/previous_points/events.html.erb

```diff
@@ -0,0 +1,38 @@
+<h5><%= link_to("Back to List", points_path, :class => 'back-link') %></h5>
+
+<% @page_title = "Attendance Points" %>
+<div class="member accomplishment index">
+<h3>Member Attendance to Events Points</h3>
+
+<p><i> Shows all the points you have earned by attending events </p></i>
+
+<p>
+	<strong> Events Points : </strong>
+	<%= @member.attendance_points(true) %>
+	<br/>
+	<strong> All Time Event Points : </strong>
+	<%= @member.accomplishment_points %>
+</p>
+
+<table class="table" summary="Member Events List">
+
+	<tr class="header">
+		<th>Name</th>
+		<th>Points</th>
+		<th>Date</th>
+		<th>Type</th>
+		<th>Details</th>
+	</tr>
+	<% @curr_semester.each do |sem| %>
+	<% @member.events.where(sem.dates === :start_time).each do |event| %>
```

Comment:
Does this work? I thought you were supposed to do:
```rb
where(semester_id: @sem_id).each
```
In fact, I didn't know that `===` was a valid Ruby operator.

Reply:
Wait, this actually won't work because of how we changed the way `Semester.current_semester` works.

Reply:
Improved this a little bit, so resolving.